proposals/fleet-multiple-abm-vpp.md

```diff
@@ -0,0 +1,130 @@
+### Multiple ABM/VPP tokens
+
+- Figma: https://www.figma.com/design/j2M1heOh8eZD6LcUJks6HE/%239956-Add-multiple-Apple-Business-Manager-and-Volume-Purchasing-Program-connections?node-id=2-130&t=3vPRYgndVgzoD8e4-0
+- Issue: https://github.com/fleetdm/fleet/issues/9956
+
+- Constraints/bussines logic:
+    - VPP tokens can be assigned to: "All teams", "No team" or a team. Default is "All teams"
+    - ABM tokens can be assigned to: "No team" or a team. Default is "No team"
```

Comment:
So currently we could say the unique ABM token is "assigned" to the configured default ABM team (which may be "no team" IIRC)? So we'll need a DB migration that will make it explicit that the existing ABM token belongs to that team, and that existing DEP-enrolled hosts we DEP-enrolled via that token?

Reply:
this is exactly right. Good catch, I added a note in the migrations section